Monika After Story/game/zz_spriteobjects.rpy

```diff
@@ -1404,6 +1509,45 @@ init -1 python:
     )
     store.mas_sprites.init_acs(mas_acs_orcaramelo_hatsune_miku_twinsquares)
 
+    ### Holly Hairclip
+    #Thanks Orca
```

Comment:
please use the commented template for ACS
in this case, this would be
```
### HOLLY HAIRCLIP
## holly_hairclip
# hairclip with a <description> on it
# thanks orca
```